pkgs/development/python-modules/bleach/default.nix

```
@@ -13,12 +13,12 @@
 
 buildPythonPackage rec {
   pname = "bleach";
-  version = "6.0.0";
+  version = "6.1.0";
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/mozilla/bleach/blob/c04958dcb931243b10e103a2e6ecfa700b190270/setup.py#L54

---

Fixed, thanks.